tools/l10n/check_locales.py

```diff
@@ -0,0 +1,40 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+#  This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+"""Check imported locales with release locales."""
+
+from os import path, listdir
+from sys import exit
+from locales import RELEASE_LOCALES
```

Comment:
Should we check for `SCREENSHOT_LOCALES` or this one?

Reply:
I'm not sure. @liuche Do you know?